travelpal/assets/js/components/friend-card.jsx

```diff
@@ -6,7 +6,7 @@ import { Row, Col } from 'reactstrap';
 import api from '../api';
 
 // Renders the details of an individual friend as a card
-export default function Friend(props) {
+export default function FriendCard(props) {
```

Comment:
I think you should name the file as you would name the component. So in this case the file name should be FriendCard.

Reply:
You want me to change the file name from `friend-card.jsx` to `FriendCard.jsx`?

Reply:
I just thought hyphenated lowercase was the name convention for `.jsx` files.

Reply:
Its just the convention I use but we can keep it as is if you want.